releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -3,7 +3,30 @@ Azure App Configuration Push extension for Azure DevOps pipeline can be installe
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
 ### v5.0.0 - February, 02 2023
-Updated the task to require Node.js 16. It previously required 10
+* **Breaking change**:
+  With the update to the task to require Node.js 16. It previously required 10. The minimum agent version for running the task on self hosted agent changed.
+
+    **Before**:
+      [Azure Pipeline agent version 2.144.0](https://github.com/microsoft/azure-pipelines-agent/releases/tag/v2.144.0) or later.
+
+    **After**:
+      [Azure Pipeline agent version 2.206.1](https://github.com/microsoft/azure-pipelines-agent/releases/tag/v2.206.1) or later.
+
+* **Breaking change**:
+
+    **Before**:

```

Comment:
The before and after is a bit hard to follow. I'd like if we can be a bit more direct on what's broken since these are often used for reference. It sounds like maybe it should be:

* Feature flag import now requires adherence to the [feature management schema](https://github.com/microsoft/FeatureManagement-Dotnet/blob/release/v3/docs/schemas/FeatureManagement.v1.0.0.json). Settings with feature flag prefix and content type will fail to import.



---

A follow up question here. Why did this break?

Reply:
Updated to have the Breaking changes with bullet point and removed the before and after.

Reply:
In version 5 of the Push task, we  integrated with the sync lib, which tries to have the import behavior consistent with the CLI, the CLI logs a warning when importing kvs, and key starts with feature flag prefix, https://github.com/Azure/azure-cli/blob/fde406f3aa731d33e08ef7a77404b25f10f3b559/src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py#L108

Reply:
The CLI logs a warning, yet this fails outright? Doesn't seem like consistent behavior.

Reply:
Since Sync client lib doesn't take a logger object,  all warnings are treated as errors in the Sync lib. Though am thinking maybe the Push task can catch this error and log it as a warning.

Reply:
The portal doesn't allow creation of feature flags as raw KVs.

Push task doesn't allow it (our current discussion)

but CLI does allow it.

I think first we need to align on what we want the behavior to be across all client side tools.

Reply:
The Portal supports importing configurations from a file and specifying the prefix as ".appconfig.featureflag/" and content type "application/vnd.microsoft.appconfig.ff+json;charset=utf-8"